homeassistant/components/zwave_js/discovery.py

```diff
@@ -1180,6 +1183,18 @@ class ZWaveDiscoverySchema:
             stateful=False,
         ),
     ),
+    # button
+    # Meter CC idle
+    ZWaveDiscoverySchema(
+        platform=Platform.BUTTON,
+        hint="meter reset",
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.METER},
+            property={RESET_METER_PROPERTY},
+            type={ValueType.BOOLEAN},
+        ),
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
This should be `EntityCategory.CONFIG` since it's an entity that allows changing something about the feature.

Reply:
Then maybe the notification idle buttons should be changed too?

Reply:
Yes, good. :+1: